packages/front-end/components/SavedGroups/ConditionGroups.tsx

```diff
@@ -133,9 +145,11 @@ export default function ConditionGroups({ groups, mutate }: Props) {
                     <SortableTH field="groupName">Name</SortableTH>
                     <SortableTH field="condition">Condition</SortableTH>
                     <th>Description</th>
+                    <th className="col-2">Projects</th>
                     <SortableTH field="owner">Owner</SortableTH>
                     <SortableTH field="dateUpdated">Date Updated</SortableTH>
-                    {(canUpdate || canDelete) && <th />}
+                    {(canUpdate({ projects: [project] }) ||
+                      canDelete({ projects: [project] })) && <th />}
```

Comment:
Conditionally rendering the `<th>` is going to cause some weird edge cases here.  If no project is selected in the left nav, but there is a saved group with projects set to `["prj_a"]`, it's possible for this `<th>` to not be rendered, but the associated `<td>` below to be rendered.

To fix, always render the `<th/>`/`<td>` no matter what and just rely on the conditional rendering inside of `<MoreMenu>`.  That should already properly handle things (if no children of MoreMenu are rendered, it should not output anything to the screen).